hilo.rb

```diff
@@ -1,2 +1,22 @@
-number = rand(1..100)
-puts number 
\ No newline at end of file
+random_number = rand(1..100)
+puts "Enter a number between 1 and 100. press enter to submit your guess"
+input = gets.chomp
+guess = input.to_i
+response = ''
+count = 0
+while count < 5
+    count = count + 1
+    response = ''
```

Comment:
I can see who you have `response` scoped here as well. But I think you can get away without using a `response` variable all together, for example in L21 you could opt for something like:
```puts (guess == random_number) ? "Correct! You win" : "Sorry, you ran out of guesses. The number was #{random_number }" ```
Its recommended to use string interpolation in most cases.